Bugsnag/Assets/Bugsnag/Runtime/Delivery.cs

```diff
@@ -73,7 +73,7 @@ public void Deliver(IPayload payload)
                         {
                             try
                             {
-                                if (!onSendErrorCallback.Invoke(report.Event))
+                                if (!RunEventCallback(onSendErrorCallback, report.Event))
```

Comment:
In the unlikely event of there being multiple callbacks that switch the handledness back and forwards, we'd still end up with `UnhandledOverridden` being set.

Given this is only called from one place, suggest you inline the logic here and store the original value before the callbacks and check it after all the callbacks.